packages/sdk/src/browser-wallets/unisat/index.ts

```diff
@@ -0,0 +1,20 @@
+import { BrowserWallet } from "../types";
+
+async function isInstalled() {
+  return false;
+}
+
+async function getAddresses() {}
+
+async function signPsbt() {}
+
+async function signMessage() {}
+
+const wallet: BrowserWallet = {
```

Comment:
not sure if this is used as a template or going to be changed in the future, but lets use the name instead of just generic `wallet` so this one can be `unisatWallet`

Reply:
In `browser-wallets/index.ts`, we export `wallet` as `unisat`.

I'm applying this pattern across all wallets, should that be changed?

Reply:
yep, so that its clearer which wallet this is without looking at the foldername